integrations/aws/client.go

```diff
@@ -128,3 +130,33 @@ func connStatus(i *integration) sdkintegrations.OptFn {
 		return sdktypes.NewStatus(sdktypes.StatusCodeError, "bad auth type"), nil
 	})
 }
+
+func connTest(i *integration) sdkintegrations.OptFn {
+	return sdkintegrations.WithConnectionTest(func(ctx context.Context, cid sdktypes.ConnectionID) (sdktypes.Status, error) {
+		vs, err := i.vars.Get(ctx, sdktypes.NewVarScopeID(cid))
+		if err != nil {
+			return sdktypes.InvalidStatus, err
+		}
+
+		cfg, err := config.LoadDefaultConfig(ctx,
+			config.WithRegion(vs.GetValueByString("Region")),
+			config.WithCredentialsProvider(
+				credentials.NewStaticCredentialsProvider(
+					vs.GetValueByString("AccessKeyID"),
+					vs.GetValueByString("SecretKey"),
+					vs.GetValueByString("Token"))),
+		)
+		if err != nil {
+			return sdktypes.InvalidStatus, err
+		}
+
+		s3Client := s3.NewFromConfig(cfg)
+
+		_, err = s3Client.ListBuckets(ctx, &s3.ListBucketsInput{})
+		if err != nil {
+			return sdktypes.NewStatus(sdktypes.StatusCodeError, "failed to connect to AWS instance"), nil
+		}
+
+		return sdktypes.NewStatus(sdktypes.StatusCodeOK, "connection test was succesfull"), nil
```

Comment:
Remove the success message in all other integrations too.

```suggestion
		return sdktypes.NewStatus(sdktypes.StatusCodeOK, ""), nil
```

Reply:
Done